draft-ietf-tls-tls13.md

```diff
@@ -3186,6 +3196,13 @@ In particular:
   that this relaxes the constraints on certificate-signing
   algorithms found in prior versions of TLS.
 
+- If the certificate_extensions list in the certificate request message
+  was non-empty, the end-entity certificate MUST contain all of the
+  specified extension OIDs that the client recognizes. For each extension
+  OID recognized by the client, all of the specified values MUST be present
+  in the client certificate. However, the client MUST ignore and skip any
+  unrecognized certificate extension OIDs.
+
```

Comment:
This text seems duplicative of the text describing the extensions. I would suggest just poingint to that section.